frontends/precinct-scanner/src/screens/election_manager_screen.tsx

```diff
@@ -126,6 +137,12 @@ export function ElectionManagerScreen({
     await unconfigure();
   }
 
+  const initialPrecinctValue = precinctSelection
+    ? precinctSelection.kind === 'AllPrecincts'
+      ? ALL_PRECINCTS_OPTION_VALUE
+      : precinctSelection.precinctId
+    : '';
```

Comment:
This is a bit of a confusing name. The `Select` is a controlled component, so its value will always be whatever is passed into the `value` prop. So this variable isn't the "initial" value but rather just the value. I might name it `precinctSelectionValue` or something like that

Reply:
Renamed.